alma/alma-installments-prestashop
# Fix data theme for gathercmsdata

### Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-2272/[-ps]-fix-data-theme-for-gathercmsdata)
For GatherCmsData, the CMS info "themes" data that we send to the Alma API is split to "theme_name" and "theme_version".

### Code changes


Split the "themes" data to "theme_name" and "theme_version", update unit tests and make compatible with Prestashop version under 1.7

### How to test

_As a reviewer, you are encouraged to test the PR locally._


On a Prestashop environnement in version under 1.7, call the URL to collect CMS data and check in the result if you have correct data.
You can also run unit test

### Checklist for authors and reviewers




### Non applicable
